components/webdriver_server/actions.rs

```diff
@@ -659,4 +650,22 @@ impl Handler {
             Ok(())
         }
     }
+
+    fn get_element_origin_relative_coordinates(
+        &self,
+        web_element: &WebElement,
+    ) -> Result<(i64, i64), ErrorStatus> {
+        let (sender, receiver) = ipc::channel().unwrap();
+        let _ = self.browsing_context_script_command(
+            WebDriverScriptCommand::GetElementInViewCenterPoint(web_element.to_string(), sender),
+        );
```

Comment:
```suggestion
        self.browsing_context_script_command(
                    WebDriverScriptCommand::GetElementInViewCenterPoint(x.to_string(), sender),
                )
                .unwrap();
```
Please revert this change. `unwrap` is ok since we would never panic at this stage.

Reply:
reverted